frontend/src/app/actions/experiments.js

```diff
@@ -4,22 +4,13 @@ export default createActions({
 
   updateExperiment: (addonID, data) => ({ addonID, data }),
 
-  fetchExperiments: (experimentsUrl, countsUrl) => {
-    let experiments = [];
-    return fetch(experimentsUrl)
-      .then(response => response.json())
-      .then(data => {
-        experiments = data.results;
-        return fetch(countsUrl);
-      })
-      .then(response => response.json())
+  fetchUserCounts: (countsUrl) => {
+    return fetch(countsUrl)
+      .then(
+        response => response.json(),
+        () => ({}))
```

Comment:
:| I'm not. This is a really good way to suppress exceptions. Can we add some error handling here?